src/simulator/short_term_object.cpp

```diff
@@ -0,0 +1,68 @@
+//========================================================================
+//  This software is free: you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License Version 3,
+//  as published by the Free Software Foundation.
+//
+//  This software is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU Lesser General Public License for more details.
+//
+//  You should have received a copy of the GNU Lesser General Public License
+//  Version 3 in the file COPYING that came with this distribution.
+//  If not, see <http://www.gnu.org/licenses/>.
+//========================================================================
+/*!
+  \file    short_term_object.cpp
+  \brief   C++ Interface: Short term object class
+  \author  Yifeng Zhu, (C) 2020
+  \email   yifeng.zhu@utexas.edu
+*/
+//========================================================================
+
+#include "simulator/short_term_object.h"
+
+ShortTermObject::ShortTermObject() {
+  // angle, (x, y)
+  pose_ = pose_2d::Pose2Df(0., Eigen::Vector2f(0., 0.));
+
+  // Example of a simple shape
+  float r = 0.5;
+  float eps = 0.001;
```

Comment:
If it's held constant it should be declared constant `constant float eps = 0.001`

Reply:
I see. I'll keep that in mind for the future code.